rust/kcl-lib/src/simulation_tests.rs

```diff
@@ -249,9 +273,9 @@ mod cube {
     }
 
     /// Test that parsing and unparsing KCL produces the original KCL input.
-    #[test]
-    fn unparse() {
-        super::unparse(TEST_NAME)
+    #[tokio::test(flavor = "multi_thread")]
+    async fn unparse() {
+        super::unparse(TEST_NAME).await
```

Comment:
Can you update the `justfile`'s template `rust/kcl-lib/tests/simtest.tmpl`?

Reply:
oh whoops sorry

Reply:
https://github.com/KittyCAD/modeling-app/pull/5853